lib/assets/stylesheets/chef/components/logo.scss

```diff
@@ -5,75 +5,205 @@ name: logo
 category: Logo
 ---
 
-Logos are delivered as configurable SVG graphics. The default orientation is vertical, 
-but you can also specify horizontal. Available for both the default and inverse [themes](../themes).
+Logos are delivered as stylable SVG graphics. The default orientation is vertical, 
+but you can also specify horizontal, provided you also supply a site name. Available 
+for both the default and inverse [themes](../themes).
 
 ```html_example
-<div class="row">
-  <div class="large-3 columns">
-    <div class="logo logo-default logo-vertical"></div>
-  </div>
-  <div class="large-4 columns">
-    <div class="logo logo-horizontal" data-tag-line="Manage"></div>
-  </div>
-  <div class="large-4 columns end">
-    <div class="logo logo-horizontal"></div>
-  </div>
-</div>
+<!-- Default -->
+<div class="logo"></div>
+
+<!-- Inverse (for dark backgrounds) -->
+<div class="logo inverse"></div>
+
+<!-- Grayscale -->
+<div class="logo grayscale"></div>
+
+<!-- Animating -->
+<div class="logo animate"></div>
+
+<!-- Just the mark -->
+<div class="logo mark"></div>
+
+<!-- Animating mark -->
+<div class="logo mark animate"></div>
 ```
+
+### Using Ruby? Have a Helper
+
+The method above uses JavaScript to retrieve the logo and place it into the DOM. If you'd 
```

Comment:
Should probably tell people to `include Chef::Web::Core::Helpers` in their ApplicationHelper or applicable place.